packages/firestore/rollup.config.node.js

```diff
@@ -45,12 +51,59 @@ export default [
     }
   },
   {
-    input: pkg['main-esm2017'],
-    output: [{ file: pkg.main, format: 'cjs', sourcemap: true }],
+    input: {
+      index: pkg['main-esm2017'],
+      bundle: path.resolve('./bundle', bundlePkg['main-esm2017'])
+    },
+    output: [
+      {
+        dir: 'dist/node-cjs',
+        format: 'cjs',
+        sourcemap: true
+      }
+    ],
     plugins: util.es2017ToEs5Plugins(),
     external: util.resolveNodeExterns,
     treeshake: {
       moduleSideEffects: false
     }
+  },
+  {
+    input: {
+      index: 'index.node.memory.ts',
+      bundle: 'index.bundle.ts'
+    },
+    output: {
+      dir: 'dist/memory/node-esm2017',
+      format: 'es',
+      sourcemap: true
+    },
+    plugins: [
+      ...util.es2017Plugins('node'),
+      replace({
+        'process.env.FIRESTORE_PROTO_ROOT': JSON.stringify('../protos')
+      })
+    ],
+    external: util.resolveNodeExterns,
+    treeshake: {
+      moduleSideEffects: false
+    }
+  },
+  {
+    input: {
+      index: path.resolve('./memory', memoryPkg['main-esm2017']),
+      bundle: path.resolve('./bundle', memoryBundlePkg['main-esm2017'])
```

Comment:
If you still have the test app, can you verify the memory build works?

It should be "@firebase/firestore/memory" and @firebase/firestore/memory/bundle"  (I believe).

I think we are still missing "memory/bundle".

Reply:
Done.